### ISSUE TITLE
Missing observers

### ISSUE THREAD

**Issue by hpmarshall** [SnowEx/snowex_db#16]
Looks like the example for distinct observers in the LayerData is missing lots of people?  If we don't use "distinct", looks like lots of "none" in the database for observers.  Not critical for hackweek at all, just wanted to create this issue for long term.  @micah-prime @micahjohnson150

**Comment by micahjohnson150** [comment]
Yep! Because of the SSA files I assumed all the Time series pits and IOP pits CSV files had the observers in the headers. Megan let me know it is only in the associated metadata. I will need to figure a good way to translate them over.
